src/java/org/apache/cassandra/cql3/statements/AlterTableStatement.java

```diff
@@ -276,6 +277,11 @@ public void validate(ClientState state)
             case DROP_COMPACT_STORAGE:
                 if (!meta.isCompactTable())
                     throw new InvalidRequestException("Cannot DROP COMPACT STORAGE on table without COMPACT STORAGE");
+                for (SSTableReader ssTableReader : Keyspace.open(keyspace()).getColumnFamilyStore(columnFamily()).getLiveSSTables())
```

Comment:
As mentioned in the PR for dtests, this only checks the version of the local sstables, but I think (I'm not very familiarized with it) that `DROP COMPACT STORAGE` will have an effect on the other nodes. I'm I missing something?